package.json

```diff
@@ -23,6 +23,7 @@
     "@blueprintjs/select": "^3.8.0",
     "@blueprintjs/table": "3.5.0",
     "@blueprintjs/timezone": "3.10.1",
+    "bfxuilib": "git+https://github.com/bitfinexcom/bfxuilib.git",
```

Comment:
This is a private repo, wont be able to be added to a public one.

Reply:
Reworked without it.